pylint/reporters/ureports/nodes.py

```diff
@@ -171,7 +183,14 @@ class Table(BaseLayout):
     * title : the table's optional title
     """
 
-    def __init__(self, cols, title=None, rheaders=0, cheaders=0, **kwargs):
+    def __init__(
+        self,
+        cols: int,
+        title: Optional[Any] = None,
```

Comment:
```suggestion
        title: Optional[str] = None,
```
Never used in actual code, but we might as well restrict it for future applications. This is based on the other classes which implement `title`.